src/components/tab-panel/tab-panel.scss

```diff
@@ -1,15 +1,20 @@
-:host(limel-tab-panel) {
-    display: block;
```

Comment:
I don't think we can remove this, did you test the tab panel in the documentation? Nothing is visible :disappointed:

Reply:
in the webclient that `display: block;` was causing it to not render for some reason. That's why I removed it. But I figured having a `height: 100%;` would show it when it's a `block`. So now this is back